sdk/textanalytics/ai-text-analytics/swagger/README.md

```diff
@@ -550,3 +472,120 @@ directive:
+    where: $.definitions.KeyPhraseTaskParameters
+    transform:
+      $["x-ms-client-name"] = "KeyPhraseExtractionAction";
+  - from: swagger-document
+    where: $.definitions.PiiTaskParameters
+    transform:
+      $["x-ms-client-name"] = "PiiEntityRecognitionAction";
+  - from: swagger-document
+    where: $.definitions.LanguageDetectionTaskParameters
+    transform:
+      $["x-ms-client-name"] = "LanguageDetectionAction";
+  - from: swagger-document
+    where: $.definitions.HealthcareTaskParameters
+    transform:
+      $["x-ms-client-name"] = "HealthcareAction";
+  - from: swagger-document
+    where: $.definitions.ExtractiveSummarizationTaskParameters
+    transform:
+      $["x-ms-client-name"] = "ExtractiveSummarizationAction";
+  - from: swagger-document
+    where: $.definitions.ExtractiveSummarizationSortingCriteria
+    transform:
+      $["x-ms-enum"].name = "ExtractiveSummarizationOrderingCriteria";
+  - from: swagger-document
+    where: $.definitions.CustomEntitiesTaskParameters
+    transform:
+      $["x-ms-client-name"] = "CustomEntityRecognitionAction";
+  - from: swagger-document
+    where: $.definitions.CustomSingleLabelClassificationTaskParameters
+    transform:
+      $["x-ms-client-name"] = "CustomSingleLabelClassificationAction";
+  - from: swagger-document
+    where: $.definitions.CustomMultiLabelClassificationTaskParameters
+    transform:
+      $["x-ms-client-name"] = "CustomMultiLabelClassificationAction";
+  - from: swagger-document
+    where: $.definitions.TaskIdentifier
+    transform:
+      $["x-ms-client-name"] = "BatchActionState";
+  - from: swagger-document
+    where: $.definitions..properties.stringIndexType
+    transform: >
+      $.description = "Specifies the measurement unit used to calculate the offset and length properties. For a list of possible values, see {@link KnownStringIndexType}.\n\nThe default is the JavaScript's default which is \"Utf16CodeUnit\".";
+```
+
+### Rename JobState
+
+```yaml
+directive:
+  - from: swagger-document
+    where: $.definitions.JobState
+    transform: $.properties.createdDateTime["x-ms-client-name"] = "createdOn";
+  - from: swagger-document
+    where: $.definitions.JobState
+    transform: $.properties.expirationDateTime["x-ms-client-name"] = "expiresOn";
+  - from: swagger-document
+    where: $.definitions.JobState
+    transform: $.properties.jobId["x-ms-client-name"] = "operationId";
+  - from: swagger-document
+    where: $.definitions.JobState
+    transform: $.properties.lastUpdateDateTime["x-ms-client-name"] = "lastModifiedOn";
+  - from: swagger-document
+    where: $.definitions.JobState
+    transform: $.properties.status["x-ms-enum"].name = "OperationStatus";
+```
+
+### Set Utf16CodeUnit as the default for StringIndexType because this is what JavaScript uses
+
+```yaml
+directive:
+  - from: swagger-document
+    where: $.definitions.StringIndexType
+    transform: $.default = "Utf16CodeUnit";
+```
+
+
+### Disable LRO
```

Comment:
And every other language will ship LROs but JS? Again, this seems like "General" section material.

Reply:
All languages are shipping LROs. This specific transform is unfortunate because the auto-generated JS LRO code is not very useful but I am not familiar with how the code generators for other language behave here so I didn't want to assume the same for them.